src/bstring.rs

```diff
@@ -44,8 +44,23 @@ pub struct BString {
 }
 
 impl BString {
+    /// Constructs a new `BString` from the given `Vec`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bstr::BString;
```

Comment:
I think the examples should include `use` statements. The one below as well.

Reply:
fixed 😄